app/Core/ExternalTask/ExternalTaskManager.php

```diff
@@ -48,6 +48,8 @@ public function getProvider($name)
     public function getProvidersList()
     {
         $providers = array_keys($this->providers);
-        return array_combine($providers, $providers);
+        if (count($providers)) {
+            return array_combine($providers, $providers);
+        }
     }
```

Comment:
Thank you for reporting this bug but you forgot a return statement here, this function must always return an array. I fixed that in master, and now a test case ensure that no regression on this code will happen in the future.